src/main/java/com/fvv/bookstore/util/OrderViewUtil.java

```diff
@@ -31,6 +34,34 @@ public static final String resolveProperValueByType(final Class clazz) {
 		return value;
 	}
 	
+	/**
+	 * Resolve proper description for a product.
+	 * 
+	 * @param product to verify.
+	 * @return the proper description for the product.
+	 */
+	public static final String resolveProperDescription(final Product product) {
```

Comment:
you can remote the final in the method signature (same for the method resolveProperValueByType)

Reply:
ok